manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VCRedist.2015+.x86
+PackageVersion: 14.34.31931

```

```suggestion
PackageVersion: 14.34.31931.0
```
Must match Control Panel.